packages/link-modules/src/definitions/publishable-api-key-sales-channel.ts

```diff
@@ -48,16 +49,5 @@ export const PublishableApiKeySalesChannel: ModuleJoinerConfig = {
         isList: true,
       },
     },
-    {
```

Comment:
question: Should this be migrate to use the sales channel module instead of removed?

Reply:
@olivermrbl what I was thinking is that there is no point going from a sales channel to an API key, which is why I removed the relationship. Let me know if you think differently

Reply:
+1
Having the graph definition and being able to traverse it from any entry point might be useful in the future.

Reply:
I don't mind adding it back, let's add it and revisit.